src/raj/FindElementResetVerify.java

```diff
@@ -0,0 +1,100 @@
+package raj;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FindElementResetVerify {
+	public static void main(String[] args) {
+	
+		System.setProperty("webdriver.chrome.driver", ".\\resources\\windows//chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("file:///G:/Technocredits/SeleniumAssignment_1.html");
+		System.out.println(driver.getTitle());
+		
+		//FirstName 
+		driver.findElement(By.id("first name")).sendKeys("Raj");
+		
+		//LastName
+		driver.findElement(By.id("last name")).sendKeys("Adatiya");
+		
+		//Email
+		driver.findElement(By.id("E-mail")).sendKeys("rajadatiya35@gmail.com");
+		
+		//CompanyName 
+		driver.findElement(By.id("Company Name")).sendKeys("IISU");
+		
+		//Gender
+		WebElement radioButtonMale = driver.findElement(By.id("maleG"));
```

Comment:
for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’   radio button and vice versa.
This validation is missing.